### ISSUE TITLE
Checkout Blocks: Review what needs to be extensible.

### ISSUE THREAD

**Issue by nerrad** [woocommerce/woocommerce-gutenberg-products-block#1216]
In this exploration, the purpose is to identify what elements of the block will need to be extensible for the MVP.

**Comment by nerrad** [comment]
I think we've already identified the two extensible pieces for now as Payment Methods (in progress) and Shipping (in progress).

I'm going to move this into the bottom of the Next Pipeline (and remove from this release) because we'll still need to evaluate potential other extensibility pieces in later iterations.

**Comment by nerrad** [comment]
Removed from Cycle 3, this is not needed for iteration 1 as we've already identified shipping and payment methods as the primary extensibile pieces for now. A review of other extensible options can happen in future iterations.

Some of the api I have planned for the checkout cart flow includes bits that should help with extensibility in other areas as well (namely exposing emitted events for extensions to hook into/react to).
